scripts/check-coding-style.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 
-script_dir=$(dirname "$(readlink -f "$0")")
-
+READLINK=$(which greadlink)
+[ -z "$READLINK" ] && {
+	READLINK=$(which readlink)
+}
+[ -z "$1" ] && {
+	echo "USAGE: $0 FILE"
+	exit 1
+}
+CHECKSOURCE=$($READLINK -f "$1")
```

Comment:
Currently only 2 files comply, so we can only check those and fail the build if an error is in those. 
Whenever another file is converted or added it can be added to the list. 
When all files were converted, we could use your find snippet to check all.